pkg/apis/nais.io/v1/naiserator_types.go

```diff
@@ -687,10 +687,28 @@ type Tracing struct {
 	Enabled bool `json:"enabled,omitempty"`
 }
 
+type Logging struct {
+	// Set this to false to disable forwarding of application logs to persistent storage.
+	// +nais:doc:Default="true"
+	Enabled bool `json:"enabled,omitempty"`
```

Comment:
Jeg antar at denne må være en boolean pointer, for å kunne skille på `enabled: false` og enabled som ikke er speccet opp.

Reply:
Ikke 100% sikker men det kan hende.

Reply:
@kimtore vil `+kubebuilder:default=true` fungere?

Reply:
Det så ut til å fungere (sjekk make generate output)

Reply:
Synes også docstringen kan kortes ned til f.eks "Enable forwarding..." i stedet for dobbel negativ :)